Code of Conduct (#1635)
nasa-gcn/gcn.nasa.gov: app/routes/_gcn.docs.code-of-conduct.md

## Retractions

If an initial automated or manually-generated Notice/Circular is later deemed to be misclassified as an astronomical object, the generating team should promptly submit another Notice/Circular informing the community of the retracted claim.

## Errata

Submitters can request edits to their own archived Circulars to correct inaccurate information, especially when the correction is important for the follow-up community's activities. These may include author lists, subject corrections, typos, and incorrect citations. This capability should not be used to add details of new observations or analysis &mdash; instead users should submit new Circulars with subjects noting "refined analysis" or "additional observations".

## Reproducing GCN Notices or Circulars in External Archives

> The GCN team welcomes cross-compatibility between community astronomical resources. If GCN Notices, Circulars, or their data are used in other systems, please link back to the original source materal at https://gcn.nasa.gov or https://gcn.gsfc.nasa.gov.

Reviewer: I think that we can be a bit more clear that GCN Circulars are scholarly publications and the normal expectations of scientific conduct for citing and reproducing them apply, since this was a point of uncertainty when we spoke with SCiMMA.

## Enforcement

> The GCN system and community aim to foster an inclusive environment for the exchange of information and ideas. Mistakes are completely understandable in these forms of rapid communication, and the GCN team is happy to work with users to correct any unintentional errors. GCN will not tolerate users who are intentionally disrespectful, misrepresent data or analysis, plagerize, or reproduce data or analysis without permission of the originator or proper citation.

Reviewer: ```suggestion
The GCN system and community aim to foster an inclusive environment for the exchange of information and ideas. Mistakes are completely understandable in these forms of rapid communication, and the GCN team is happy to work with users to correct any unintentional errors. GCN will not tolerate users who are intentionally disrespectful, misrepresent data or analysis, plagiarize, or reproduce data or analysis without permission of the originator or proper citation.
```

## Enforcement

The GCN system and community aim to foster an inclusive environment for the exchange of information and ideas. Mistakes are completely understandable in these forms of rapid communication, and the GCN team is happy to work with users to correct any unintentional errors. GCN will not tolerate users who are intentionally disrespectful, misrepresent data or analysis, plagiarize, or reproduce data or analysis without permission of the originator or proper citation.

Circulars submitting privileges are granted for new users via the GCN [peer endorsement system](/user/endorsements). Any user found to intentionally violate these policies will have their Circulars/Notices submission privileges revoked. This may lead to a review of the peer endorser who approved the offending user.

## Acknowledgement

> The GCN team requests that any presentation, publication, or document that mentions the GCN system, specific GCN Circulars or Notices, to please reference the General Coordinates Network (https://gcn.nasa.gov) and cite Circulars using bibliographic records from the [SAO/NASA ADS](https://ui.adsabs.harvard.edu).

Reviewer: ```suggestion
The GCN team requests that any presentation, publication, or document that mentions the GCN system, specific GCN Circulars or Notices, to please reference the General Coordinates Network (https://gcn.nasa.gov) and cite Circulars using bibliographic records from the [SAO/NASA Astrophysics Data System (ADS)](https://ui.adsabs.harvard.edu).
```

## Acknowledgement

> The GCN team requests that any presentation, publication, or document that mentions the GCN system, specific GCN Circulars or Notices, to please reference the General Coordinates Network (https://gcn.nasa.gov) and cite Circulars via [SAO/NASA ADS](https://ui.adsabs.harvard.edu).

Reviewer: ```suggestion
The GCN team requests that any presentation, publication, or document that mentions the GCN system, specific GCN Circulars or Notices, to please reference the General Coordinates Network (https://gcn.nasa.gov) and cite Circulars using bibliographic records from the [SAO/NASA ADS](https://ui.adsabs.harvard.edu).
```
